pkg/ccl/crosscluster/physical/stream_ingestion_dist_test.go

```diff
@@ -54,25 +54,6 @@ func TestMeasurePlanChange(t *testing.T) {
 		return plan
 	}
 
-	getNodes := func(plan *sql.PhysicalPlan) (src, dst map[string]struct{}, nodeCount int) {
```

Comment:
in a follow up, non-backportable commit, we should move this test to the shared cross cluster package and test both the LDR and PCR getNodes implementations.

Reply:
will do